docs/processes/release/tasks/release-build.md

```diff
@@ -0,0 +1,261 @@
+
+One Windows system, path separator back slash '\\' should be replaced with
+forward slash '/'. For example:
+
+`java7.bootclasspath=C:/jdk/oracle/7u80/jre/lib/resources.jar;C:/jdk/oracle/7u80/jre/lib/rt.jar;C:/jdk/oracle/7u80/jre/lib/sunrsasign.jar;C:/jdk/oracle/7u80/jre/lib/jsse.jar;C:/jdk/oracle/7u80/jre/lib/jce.jar;C:/jdk/oracle/7u80/jre/lib/charsets.jar;C:/jdk/oracle/7u80/jre/lib/jfr.jar;C:/jdk/oracle/7u80/jre/classes`
+
+#### Additional Configuration for GA
+
+*   Add 2 more properties to `build-local.properties`
+    *   `maven-ant-tasks.jar` - Specify the path to the Maven Ant Tasks jar
+        file.
+    *   `gpg.user` - Specify the ICU Project PGP key user name
+
+> For example, build-local.properties look like below:
+
+> `java7.bootclasspath=/home/yoshito/jdk/oracle/jdk1.7.0_80/jre/lib/resources.jar:/home/yoshito/jdk/oracle/jdk1.7.0_80/jre/lib/rt.jar:/home/yoshito/jdk/oracle/jdk1.7.0_80/jre/lib/sunrsasign.jar:/home/yoshito/jdk/oracle/jdk1.7.0_80/jre/lib/jsse.jar:/home/yoshito/jdk/oracle/jdk1.7.0_80/jre/lib/jce.jar:/home/yoshito/jdk/oracle/jdk1.7.0_80/jre/lib/charsets.jar:/home/yoshito/jdk/oracle/jdk1.7.0_80/jre/lib/jfr.jar:/home/yoshito/jdk/oracle/jdk1.7.0_80/jre/classes`
+
+> `maven-ant-tasks.jar =
+> /home/yoshito/java-libs/maven-ant-tasks/maven-ant-tasks-2.1.3.jar`
+> `gpg.user = ICU Project`
+
+> *Note: Maven Ant Tasks is no longer maintained. ICU team should find out
+> another method to publish ICU artifacts to the repository
+> (ticket[#12811](http://bugs.icu-project.org/trac/ticket/12811))*
+
+*   Create maven local setting file `$HOME/.m2/settings.xml` if absent. Then add
+    the Sonatype Nexus Repository Manager account information in the file. For
+    example:
+
+> `<settings>`
+
+> `<servers>`
+
+> `<server>`
+
+> `<id>icu4j-releases</id>`
+
+> `<username>yumaoka</username>`
+
+> `<password>OSSRH-SonaType-Password</password>`
+
+> `</server>`
+
+> `<server>`
+
+> `<id>icu4j-snapshots</id>`
+
+> `<username>yumaoka</username>`
+
+> `<password>OSSRH-SonaType-Password</password>`
+
+> `</server>`
+
+> `</servers>`
+
+> `</settings>`
+
+#### Building Release Files for Milestones/Release Candidates
+
+> 1. Checkout tagged version - Use clean checkout of Git (SVN up to ICU62) tag
+> (e.g. release-61-0-1, release-61-rc..) for building release files.
+
+> 2. Run `$ ant releaseVer` - This build target generates ICU release files with
+> version number in a folder `release `under the ICU4J root directory.
+
+> > This ant target prints out warning messages as below if the build system is
+> > not properly configured.
+
+> > `_verify_config_for_release:`
```

Comment:
some of the sample outputs and configs etc. have funky formatting but are readable...

Reply:
Done.